lib/apiservers/portlayer/swagger.yml

```diff
@@ -1082,12 +1082,14 @@ definitions:
     properties:
       containerId:
         type: string
-      imageId:
+      layerId:
         type: string
-      created:
+      repoName:
         type: string
-        format: datetime
-      name:
+      created:
+        type: integer
+        format: int64
```

Comment:
What's the interpretation of this date value?  Or better yet, how do I turn it into a DateTime?